### DISCUSSION TITLE
[2190290]不双修就去世 本体+dlc

### DISCUSSION THREAD

**Discussion by fengse2020** [pjy612/SteamManifestCache#78] [Share]
appid:2190290
code:8EFFA49AEB31E1CE
email:2646331095@qq.com
[不双修就去世.zip](https://github.com/pjy612/SteamManifestCache/files/13696131/default.zip)

**Comment by pjy612** [comment]
查收

**Comment by millerzhang02-art** [comment]
密码则么不对
